plugins/single_plugins/fisheye.cpp

```diff
@@ -10,7 +10,8 @@
  * copies of the Software, and to permit persons to whom the Software is
  * furnished to do so, subject to the following conditions:
  *
- * The above copyright notice and this permission notice shall be included in all
+ * The above copyright notice and this permission notice shall be included in
+ * all
```

Comment:
This looks wrong..

Reply:
it is in column 81 after all :)

But good point, this looks important enough to increase the comment width to 80.

Reply:
Well actually we can just rewrap the comment appropriately - the MIT license is originally wrapped at 80 columns, but because it is in a comment here we have a few additional spaces in front, so it should be wrapped even more.